06-release-engineering-analytics.Rmd

```diff
@@ -373,19 +373,19 @@ defined challenges include keeping everyone informed and engaged, monitoring cha
 setting priorities in cross-project coordination.
 
 @laukkanen2018a has described what effects modern release engineering have on software with 
-different organizational contexts. In this, this paper specifically focusses on continuous 
-deployment (CD) practices. The paper has found that high internal quality standards combined 
-with the large distributed organizational context of large corporations slowed the 
-verification process down and therefore had a negative impact on release capability. While in 
-small corporations, the lack of internal verification measures due to a lack of resources was 
+different organizational contexts. This study specifically focusses on continuous deployment 
+(CD) practices. The paper has found that high internal quality standards combined with the 
+large distributed organizational context of large corporations slowed the verification process 
+down and therefore had a negative impact on release capability. However, in small 
+corporations, the lack of internal verification measures due to a lack of resources was 
 mitigated by code review, disciplined CI and external verification by customers in customer 
-environments. More about the factors that can play a role in @rodriguez2017a, where an 
-overview of contributing factors in continuous deployment are defined and categorized based on 
-literature between 2001 and 2014.
+environments. More about the factors that can play a role are addressed by @rodriguez2017a, 
+where an overview of contributing factors in continuous deployment are defined and categorized 
```

Comment:
Maybe better: "... addressed by @rodriguez2017a, who define and categorize an overview of contributing factors in continuous deployment on literature between 2001 and 2014."

Reply:
Wow yeah sorry that sentence doesn't make any sense